$DragonFly: src/lib/libarchive/patches/Attic/archive_platform.h.patch,v 1.4 2007/03/26 00:03:44 pavalos Exp $
--- archive_platform.h.orig	2007-03-25 17:03:13.000000000 -0400
+++ archive_platform.h	2007-03-25 17:03:35.000000000 -0400
@@ -38,7 +38,7 @@
 
 #if defined(HAVE_CONFIG_H)
 /* Most POSIX platforms use the 'configure' script to build config.h */
-#include "../config.h"
+#include "config.h"
 #elif defined(__FreeBSD__)
 /* Building as part of FreeBSD system requires a pre-built config.h. */
 #include "config_freebsd.h"
